gitax/Visual Studio Projects/C Sharp Projects/GTXLibGit2Sharp/GTXLibGit2Sharp.csproj

```diff
@@ -84,17 +85,33 @@
     <Compile Include="Properties\AssemblyInfo.cs" />
   </ItemGroup>
   <ItemGroup>
+    <AXProxy Include="Class.Global.axproxy">
+      <Generator>MSBuild:Compile</Generator>
+    </AXProxy>
     <AXProxy Include="Table.SysVersionControlTmpItem.axproxy">
       <Generator>MSBuild:Compile</Generator>
     </AXProxy>
   </ItemGroup>
+  <ItemGroup />
+  <ItemGroup>
+    <Content Include="lib\git2-e0902fb.dll" />
+    <Content Include="lib\LibGit2Sharp.dll" />
+    <Content Include="LibGit2Sharp.xml">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+    </Content>
+  </ItemGroup>
```

Comment:
yes. When you build the project, it will run msbuild on your project file including the code above. You do have to restart the client though after that, but that's a small price.